libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -525,6 +525,7 @@ bool PluginManager::DropFile(const wxString &fileName)
                   mRegisteredPlugins[id].SetEnabled(enable);
                // Make changes to enabled status persist:
                this->Save();
+               this->NotifyPluginsChanged();
```

Comment:
Not reachable with default Experimental.cmake but it's good you noticed it

Reply:
I see one use of Save() not followed by the publication, and that is in `PluginManager::InitializePlugins()`.  Is that right or wrong?

Reply:
Not wrong, I see, because the only subcriber is RealtimeEffectListWindow which cannot possibly be constructed earlier than that initialization.